modules/custom/social_graphql/tests/src/Kernel/SocialGraphQLTestBase.php

```diff
@@ -43,7 +44,9 @@ protected function setUp() : void {
     $this->installConfig("social_graphql");
 
     // Set up the schema and use the Open Social GraphQL server in queries.
-    $this->server = Server::load("open_social_graphql");
+    $server = Server::load("open_social_graphql");
+    self::assertInstanceOf(ServerInterface::class, $server, "Test set-up failed: 'open_social_graphql' server is not installed.");
```

Comment:
It might feel pedantic that PHPStan complains about the assignment to `$this->server` but it helps us for two reasons.

1. Drupal's `Server::load` falls through to `EntityInterface::load` whose return type is `EntityInterface`. So from a type annotation perspective we're not actually guaranteed a `Server` (or `ServerInterface`) instance. We all know that's how Drupal works, but that's only if implementations are well behaved.
2. Our test would break if `open_social_graphql` was not installed and `Server::load` returned NULL. This could show up as a "Can't assign NULL to non-nullable object" or "call to function on NULL" depending on the type of `$this->server`. Neither are particularly helpful. An explicit check would tell a developer that their test setup has been broken.